modules/icinga/manifests/check/graphite.pp

```diff
@@ -100,9 +100,9 @@
   }
 
   if $action_url == undef {
-
     if $::aws_migration {
-      $app_domain = hiera('app_domain_internal')
+      # temporary solution while we have two Icinga instances
+      $app_domain = "blue.${::aws_environment}.govuk.digital"
```

Comment:
I don't think Graphite requires the `blue.` part, so it might be good to omit it to avoid any questions about what it means...

Reply:
Good idea, I've amended the commit to remove the `blue.`.